services-custom/s3-transfer-manager/src/main/java/software/amazon/awssdk/transfer/s3/internal/DownloadDirectoryHelper.java

```diff
@@ -197,37 +197,6 @@ private CompletableFuture<CompletedFileDownload> doDownloadSingleFile(DownloadDi
         }
     }
 
-    /**
-     * If the prefix is not empty AND the key contains the delimiter, normalize the key by stripping the prefix from the key.
-     *
-     * If a delimiter is null (not provided by user), use "/" by default.
-     *
-     * For example: given a request with prefix = "notes/2021"  or "notes/2021/", delimiter = "/" and key = "notes/2021/1.txt",
-     * the normalized key should be "1.txt".
-     */
-    private static String normalizeKey(ListObjectsV2Request listObjectsRequest,
```

Comment:
Minor: it'd be great if we can create a separate commit for refactoring in the future since it's hard to tell the diff and easy to miss things if we do refactoring (extracting this method out of this class) and fixing in the same commit. It'd also make reviews easier.

Reply:
will keep that in mind for future pr